demo-app/src/app/ot/playground/[slug]/page.tsx

```diff
@@ -0,0 +1,37 @@
+import * as COMPONENTS from '@/open-truss/components'
+import { applyConfiguration, parseYaml } from '@open-truss/open-truss'
+import { promises as fs } from 'fs'
+import { notFound } from 'next/navigation'
```

Comment:
This is a very interesting addition. I had wanted to move this file essentially into the OT package, but this would mean that OT depends on `next` and I don't think we want that 🤔